### PR TITLE
JIRA: CB-16641 ; Add telemetry data collection to the datalake resize flow

### PR DESCRIPTION
**Pull request by RokolyA94**
JIRA: https://jira.cloudera.com/browse/CB-16641

Add telemetry data collection to the datalake resize flow.

### CODE REVIEW

**Comment by oleewere** [comment]
is it right to remove request ids?

**Comment by horadla23** [comment]
something is wrong with this PR, requestId removal should not have been shown, it is already in master with this commit: https://github.com/hortonworks/cloudbreak/commit/7d1eb1ffeeab2168bf2400c538565ab785f7bb5e

**Comment by horadla23** [comment]
probably a rebase went wrong or similar

**Comment by lnardai** [comment]
@RokolyA94 
Are you planning to fix the PR content or should we close this?

**Comment by RokolyA94** [comment]
> @RokolyA94 Are you planning to fix the PR content or should we close this?

Yes. Soon. I was on holiday sorry.

File: usage-collection/src/generated/main/java/com/cloudera/thunderhead/service/common/usage/UsageProto.java [hunk 1]
```diff
@@ -246059,575 +246110,577 @@ public com.cloudera.thunderhead.service.common.usage.UsageProto.RMUsageEventSour
...
-      "T_IN_PROGRESS\020\016\022\023\n\017RESET_COMPLETED\020\017\022\020\n\014" +
-      "RESET_FAILED\020\020\022\014\n\010ORPHANED\020\021\022\030\n\024ORPHANED" +
-      "_NOT_ENABLED\020\022\022\016\n\nUNORPHANED\020\023\022\025\n\021UPGRAD" +
-      "E_REQUESTED\020\024\022\022\n\016UPGRADE_FAILED\020\025\022\025\n\021UPG" +
-      "RADE_COMPLETED\020\026\022#\n\037UPGRADE_K8S_VERSION_" +
-      "IN_PROGRESS\020\027\022 \n\034UPGRADE_WORKLOAD_IN_PRO" +
-      "GRESS\020\030\022\030\n\024ROLLBACK_IN_PROGRESS\020\031\022#\n\037UPG" +
-      "RADE_DEPLOYMENTS_IN_PROGRESS\020\032\"\232\001\n\025CDPDF" +
-      "OperationDetails\022\n\n\002id\030\001 \001(\t\022\013\n\003crn\030\002 \001(" +
-      "\t\022\021\n\taccountId\030\003 \001(\t\022\022\n\nworkflowId\030\004 \001(\t" +
```
Comment (kkalvagadda1) [inline usage-collection/src/generated/main/java/com/cloudera/thunderhead/service/common/usage/UsageProto.java:246242]: Is this an auto-generated change?

**Review by kkalvagadda1 — COMMENTED** [review]
How were the changes tested?

**Comment by horadla23** [comment]
now the commit message is not correct once again

**Review by bergerdenes — APPROVED** [review]

**Pull request merged**